trady_functions.lua

```diff
@@ -661,6 +662,209 @@ function StringToBool( inString )
 end
 
 
+--Functions exported from handy--
+
+function GetChestHeightCheat( inChest )
+	local chestGrid = inChest:GetContents()
+	LOGINFO( "This function serves no purpose now! You should consider chest:GetContents():GetHeight() now!" )
+	LOGINFO( "Also, you might find Handy's new 'IsChestDouble()' useful for your case" )
+	return chestGrid:GetHeight()
+end
+function IsChestDouble( inChest )
+	local chestHeight = inChest:GetContents():GetHeight()
+	if( chestHeight == 3 ) then
```

Comment:
This will not work as expected. All cChestEntity objects are single-chests. The server combines the chests only when their UI is opened by a player.